serverless/containers/reference-content/configure-trigger-inputs.mdx

```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: Considerations to configure event retention for SQS trigger inputs
+  description: This page explains how to configure event retention for SQS trigger inputs
+content:
+  h1: Considerations to configure event retention for SQS trigger inputs
+  paragraph: This page explains how to configure event retention for SQS trigger inputs
+tags: containers
+dates:
+  validation: 2023-10-20
+  posted: 2023-06-09
+categories:
+  - serverless
+  - containers
+---
+
+<Message type="important">
+  This feature is currently in beta and only allows a maximum of 10 inflight requests. Containers that are configured with more than 10 replicas will not be used at full capacity. This limitation will be removed in the future.
+</Message>
+
+Triggers get events from an input, such as an SQS queue, and forward them to a container, which will scale up according to its settings to accommodate the workload. This process uses back pressure, so that no new events are read if the container instances have not yet finished processing the previous ones.
+
+<Message type="note">
+  Triggers only keep a buffer of the messages that are inflight, they do not drain all the messages of the input in advance.
+</Message>
+
+As a result, in some scenarios such as event bursts or slow computations, events may stay in the input buffer for a while before being consumed. If the input messages in the queue are set with a timeout, such as the  [MessageRetentionPeriod](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SetQueueAttributes.html) in SQS queues, events may be deleted before triggering the container.
+
+The implementation of the core trigger behavior is input agnostic, it is therefore your responsibility to configure the input buffers according to your use case to avoid losing events.
```

Comment:
```suggestion
The implementation of the core trigger behavior is input-agnostic. It is therefore your responsibility to configure the input buffers according to your use case to avoid losing events.
```